extensions/azure/processors/PutAzureBlobStorage.h

```diff
@@ -77,11 +77,8 @@ class PutAzureBlobStorage final : public AzureStorageProcessorBase {
         return -1;
       }
 
-      result_ = blob_storage_wrapper_.uploadBlob(blob_name_, buffer.data(), flow_size_);
-      if (!result_) {
-        return read_ret;
-      }
-      return result_->length;
+      result_ = azure_blob_storage_.uploadBlob(params_, gsl::span<const uint8_t>{buffer.data(), flow_size_});
```

Comment:
Specifying the span template argument should be unnecessary. CTAD may work, but `gsl::make_span` definitely does.

Reply:
Updated in 32d6542825310892eed3e5648dd55bc5b0d66257